src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -90,6 +90,8 @@ public function addConfiguration(NodeBuilder $node): void
         $node
             ->arrayNode($this->getKey())
                 ->fixXmlConfig($this->getKey())
+                ->fixXmlConfig('issuer')
+                ->fixXmlConfig('algorithm')
```

Comment:
This one can only work in 8.0 as it already exists in 7.4 (not even sure that this is a good idea as the semantic is not going to be the same between the 2 versions).

Reply:
See
* https://github.com/symfony/symfony/pull/61702